stream-api/src/entities/channel.entity.ts

```diff
@@ -0,0 +1,30 @@
+import { Track } from './track.entity';
+import {
+  Entity,
+  Column,
+  PrimaryGeneratedColumn,
+  JoinColumn,
+  OneToMany,
+  OneToOne,
+} from 'typeorm';
+
+@Entity('channel')
+export class Channel {
+  @PrimaryGeneratedColumn('increment')
+  id: number;
+
+  @Column()
+  name: string;
+
+  @Column()
+  thumbnail: string;
+
+  @Column()
+  description: string;
+
+  @OneToMany(() => Track, (tracks) => tracks.channel)
+  tracks: Track;
```

Comment:
do we need to have tracks with channel?

Reply:
Yes we will need it. Every channel will have its separate tracks and we will call them accordingly from the api